packages/components/controlled-input/src/ControlledInput.tsx

```diff
@@ -8,13 +8,15 @@ import React, {
   useRef,
   useEffect,
 } from 'react';
-import cn from 'classnames';
+import { cx } from 'emotion';
 import { Done, Minus } from '@contentful/f36-icons';
 import type { IconProps } from '@contentful/f36-icons';
 
-import styles from './ControlledInput.css';
+import { styles } from './ControlledInput.styles';
+import { Box } from '@contentful/f36-core';
 
-export interface ControlledInputProps extends HTMLProps<HTMLInputElement> {
+export interface ControlledInputProps
```

Comment:
You will want to extend the types from the box component as well, it gives you the shared types like `className` and `testId`.

Reply:
I imagine colliding props. Should I have a separate `boxProps: BoxProps` ?